src/main/java/edu/mum/sonet/controllers/rest/PostControllerRest.java

```diff
@@ -46,9 +46,12 @@ private User getCurrentUser() throws UsernameNotFoundException {
             consumes = {"multipart/form-data"})
     public Post save(@Valid Post post, HttpServletRequest request) {
         String rootDirectory = request.getSession().getServletContext().getRealPath("/");
-        String contentUrl = fileService.saveFile(post.getContentFile(), rootDirectory + "post/");
-        contentUrl = contentUrl.substring(contentUrl.lastIndexOf("post/"));
-        post.setContentUrl("/" + contentUrl);
+        if(post.getContentFile() != null) {
```

Comment:
I believe @yadirhb removed this class in his PR.. Make sure this isn't a problem.

Reply:
i added it because i tried to add post it throws an exception if there is no content, if @yadirhb  remove this class it's fine, he actually handle this problem on his branch